README.md

```diff
@@ -16,50 +24,52 @@ The crawl results can be output in 3 different modes:
 
 ### Setup
 
+In order to index crawl results into an Elasticsearch instance, you must first have one up and running.
```

Comment:
```suggestion
In order to index crawl results into an Elasticsearch instance, you must first have one up and running.
```

This sentence is a little confusing, does "one" refer to Elasticsearch here. If yes, then consider calling this a prerequisite. 

I would also consider making **Connecting to Elasticsearch** part of the integrated overall setup flow, rather than appending it to end of page.

Reply:
(after reading further if we have a prerequisites section saying need running Elasticsearch instance then we could safely drop this)